tensorflow/lite/experimental/micro/kernels/cmsis-nn/depthwise_conv.cc

```diff
@@ -206,7 +207,15 @@ TfLiteStatus EvalQuantizedPerChannel(TfLiteContext* context, TfLiteNode* node,
         ARM_MATH_SUCCESS);
   }
 #else
-#error ARM_MATH_DSP and ARM_MATH_LOOPUNROLL must be set
+
```

Comment:
Actually, it would be good for the user to be notified that there's no optimization for this target. You can add a directive for that. For example:
#pragma message("CMSIS-NN optimization for depthwise_conv not available for this target. Using ref kernel.")

Reply:
done! Please do review